### DISCUSSION TITLE
1.8.4

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#1008] [Обновления]
## Список изменений

- Добавлена кнопка "Сообщить об ошибке" в меню расширения
- Исправлена ошибка из-за которой кнопка перевода могла не пропадать на мобильных устройствах (#1005)
- Исправлена ошибка из-за которой в некоторых случаях нельзя было свайпнуть экран при воспроизведение видео на мобильных устройствах (#1006)

## What's Changed
* Update 1.8.4 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/1007

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.8.3...1.8.4

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.8.4'>1.8.4</a>.</em>

**Comment by DanDiablo83** [comment]
Доброго времени суток! Я извиняюсь, можно ли сделать\исправить для vivaldi? 
Берется версия для гуглхрома, но есть ошибки.
![screen4](https://github.com/user-attachments/assets/3a8b5dc7-f3c7-405e-a93f-27b7e3b5069a)

**Reply by ilyhalight** [reply]
не вижу тут ошибок от расширения, проверьте, что у вас расширение активно и включен режим разработчика

**Reply by DanDiablo83** [reply]
Благодарю! Не знал, что нужно включить режим разработчика, в Firefox все работает без него. С новым годом!
Пусть Змея несет удачу,
Позитив, добро в придачу,
В двадцать пятый Новый год
Чудо пусть произойдет!

Чтобы радость и веселье
Поднимали настроение,
Ёлка, шарики, салют,
И куранты уже бьют!

С Новым годом, с Новым счастьем,
Пусть он будет без ненастья,
Мирным, светлым, ярким, клёвым,
Самым лучшим, самым Новым!

**Comment by FiLinX** [comment]
ребят, а помогите мне а?
если не затруднит, скиньте файл аудиодорожки перевода вот с этого видео - https://youtu.be/lcjdwSY2AzM 
я не знаю, что у меня не так, но вот именно на этом видосе у меня глючит скрипт..
я по нажатию на панель VOTа - мгновенно вижу типа получение перевода - на панельке становится синеньким Отключить, и по стрелочке даунлода скачивает типа mp3 дорожку (но на деле скачивается пустая обманка с весом 4072 байта)
и естесственно никакого перевода при запуске воспроизведения нет, лишь в выставленный уровень приглушается звук оригинала..
вот что я вижу в консоли после типа получения перевода - https://a.i.getapic.me/li9j.jpeg 

я конечно проверял с выставлением и авто и энглишь, и включал отключал аудио-плеер другой  -  но ничего не помогает..
притом всё нормально с получением перевода и воспроизведением на любых других иноземских роликах..
вот такая странная трабла..

добавлю ещё:
при снятии галочки с "загружать файл с названием видео" и клике по стрелочке даунлода, перехожу на адрес:
https://vtrans.s3-private.mds.yandex.net/tts/prod/cdd72ac50fd54ef0aa35981b513cd867.mp3?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=WjGREUc47LBUFZn6pllC%2F20250416%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20250416T132735Z&X-Amz-Expires=7200&X-Amz-SignedHeaders=host&X-Amz-Signature=3bf2aca2adff54ed3269ed953af7f54b71cb181e26ff81c186e212b359ea120e
с текстом на странице:
<Error>
<script/>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Resource>/tts/prod/cdd72ac50fd54ef0aa35981b513cd867.mp3?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=WjGREUc47LBUFZn6pllC%2F20250416%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20250416T132735Z&X-Amz-Expires=7200&X-Amz-SignedHeaders=host&X-Amz-Signature=3bf2aca2adff54ed3269ed953af7f54b71cb181e26ff81c186e212b359ea120e</Resource>
<RequestId>01e6d823814eba45</RequestId>
</Error>
.onkcffiihkfgojhpdpgimkinjfmgmobj{cursor:hand;display:inline !important;}

**Reply by ilyhalight** [reply]
> The specified key does not exist.
> /tts/prod/cdd72ac50fd54ef0aa35981b513cd867.mp3?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=WjGREUc47LBUFZn6pllC%2F20250416%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20250416T132735Z&X-Amz-Expires=7200&X-Amz-SignedHeaders=host&X-Amz-Signature=3bf2aca2adff54ed3269ed953af7f54b71cb181e26ff81c186e212b359ea120e
> 01e6d823814eba45

У яндекса что-то баганулось и из-за этого сервер отдает невалидную ссылку на файл для англ и авто языков. Можно поставить например французский язык, и там уже будет нормальное аудио

> и если можно, подскажите плиз ещё раз - что нужно подредактировать в скрипте, чтобы он установился в темпермонку как ещё один новый скрипт, а не предлагал замену-обнову уже стоящего другой версии?

В строчке `@namespace vot` добавьте что-нибудь к `vot` или напишите вместо него что-нибудь другое

**Reply by FiLinX** [reply]
> Можно поставить например французский язык, и там уже будет нормальное аудио

уух..  работает)
до чего ж удивителен этот мир  -  постоянно вот натыкаюсь на ситуации, когда хочется возгласить "а что так можно было?"  ))
Спасибо!
и за @namespace тоже! )

**Comment by FiLinX** [comment]
и если можно, подскажите плиз ещё раз  -  что нужно подредактировать в скрипте, чтобы он установился в темпермонку как ещё один новый скрипт, а не предлагал замену-обнову уже стоящего другой версии?
я помню, что вы уже подсказывали мне это, и я успешно применял, но вот прям как веником с башки вымело, и не помню - что именно нужно изменять((

**Comment by chipollinch** [comment]
в мaнки(beta) скрипт(1.10.5) не открывается в vivаldi(7.7.3851.58), mv2 и оригинал открывается но нe пepeвoдит.
